docusaurus/docs/advanced-configuration.md

```diff
@@ -3,21 +3,17 @@ id: advanced-configuration
 title: Advanced Configuration
 ---
 
-The `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a plugin.
+The plugin `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a Grafana plugin. Although you can make changes, we recommend against doing so. Instead, follow the guidance in this topic to customize your tooling configs.
```

Comment:
Are we sure we need the word "plugin" here before ".config/"? (I think the .config directory doesn't closely relate to the plugin "itself", it is more like generated build configuration, so the plugins can be packaged by the plugin devs with no effort.)

Reply:
Fixed.